crates/bevy_input/src/gamepad.rs

```diff
@@ -1661,15 +1770,21 @@ mod tests {
 
     #[test]
     fn test_button_axis_settings_default_filter() {
+        // new (raw), old (scaled), expected new (scaled)
         let cases = [
+            // high enough to be rounded to 1
             (1.0, None, Some(1.0)),
             (0.99, None, Some(1.0)),
             (0.96, None, Some(1.0)),
             (0.95, None, Some(1.0)),
-            (0.9499, None, Some(0.9499)),
-            (0.84, None, Some(0.84)),
-            (0.43, None, Some(0.43)),
-            (0.05001, None, Some(0.05001)),
+            // for the following, remember that 0.05 is the "low" value and 0.95 is the "high" value
```

Comment:
most tests are just testing the scaling, so it makes more sense to hardcode the expected result